packages/wallet/src/redux/protocols/advance-channel/reducer.ts

```diff
@@ -154,6 +155,7 @@ function initializeWithNewChannel(
       nextParticipant(participants, ourIndex),
       processId,
       [signResult.signedCommitment],
+      ADVANCE_CHANNEL_PROTOCOL_LOCATOR,
```

Comment:
Shouldn't this be `protocolLocator` from state?

Reply:
From what state?

Reply:
It should be the`protocolLocator` variable that is passed into `initialize`. That way the action that gets dispatched has the full `protocolLocator` string (IE:`DirectFunding-AdvanceChannel`) as opposed to just `AdvanceChannel.